NREL/ThermalNetwork
# Documentation site for ThermalNetwork

#### Any background context you want to provide?
This is a simple documentation interface, using [the Material theme for MkDocs](https://squidfunk.github.io/mkdocs-material/getting-started/).

Using Mkdocs enables us to work in markdown and avoid the complication of sphinx/rst. 
#### What does this PR accomplish?
This publishes documentation to https://nrel.github.io/ThermalNetwork/ via a [GitHub Actions workflow](https://github.com/NREL/ThermalNetwork/blob/mkdocs/.github/workflows/mkdocs.yml#L1) when changes to the [docs](https://github.com/NREL/ThermalNetwork/tree/mkdocs/docs) folder are merged to `main` branch. All docs are written entirely in markdown and there is a yaml config file in the project root.
#### How should this be manually tested?
When developing, we can [serve docs locally](https://squidfunk.github.io/mkdocs-material/creating-your-site/#previewing-as-you-write) and it updates the live draft with every save.
1. Check out this branch
2. `mkdocs serve`
3. Browse to `http://127.0.0.1:8000/`

There is a LOT of customization we can do with [MkDocs](https://www.mkdocs.org/user-guide/) - this is very basic right now. 
#### What are the relevant tickets?

#### Screenshots (if appropriate)
